### ISSUE TITLE
Grading Overview - display mark name on dot hover

### ISSUE THREAD

**Issue by PandorasFox** [Submitty/Submitty#3007]
Currently in the middle of marking exam 2 - upon needing to switch another question (problem 2), it's hard to tell which dot is which mark (and where I should start grading, since I can't tell which dots are which mark).

The most ideal thing for users would be a small popup on hover. This can be done through a couple ways:

* embed some html after each dot. This looks like it might be harder to do, since the dots are just `<i>` tags that are styled with css.

* embed the mark name in the dot's node (i.e. `<i class="fa fa-circle grader-3" data-mark-name="Problem 1.1: Example Problem"></i>` and then bind some js to the `onmouseover` events for each of the `.i.fa.fa-circle` nodes to display a hover popup (or maybe give them all an extra class to make selecting a bit easier. I dunno.)

This would be very ideal since it'll make it easier to see _what_ is at what point, since we usually talk about entire problems (1, 2, 3, 4) but the marks are for each subproblem, so it's hard to map things 1:1 unless it's the first or last mark.

**Comment by VictorLi718** [comment]
I am not quite sure which part is your issue in. Is these green and empty dots? Could you please explain more and provide with a screenshot?
![image](https://user-images.githubusercontent.com/30506009/47613259-265bca00-da62-11e8-9e42-19fc309fe98c.png)

**Comment by PandorasFox** [comment]
I can't really give a screenshot right now; I can make a mock-up later.

Basically - imagine we have a rubric with the following components:

* Cover sheet
* Question 1.1
* Question 1.2
* Question 1.3
* Question 2
* Question 3.1
* Question 3.2, 3.3
* Question 4
* Question 5.1
* Question 5.2

Let's say I was working on grading question 4. After a few hours, that question might be completely graded and everyone who was working on it will switch to another problem - let's say question 2.

We have no indicator at all for which people haven't had that component graded yet - I have no way of knowing which dot corresponds to which component since one dot does not map to one question.

Basically the problem is that there's no way to easily see which students are ungraded for a certain component (and then go grade them). The easiest way to resolve that is to add some way to display the component name when a dot is hovered.

**Comment by bmcutler** [comment]
closing because it has already been implemented...

**Comment by PandorasFox** [comment]
And I’ll never get to see it :’(

**Comment by bmcutler** [comment]
grad school?

**Comment by PandorasFox** [comment]
Not any time soon :P
